advanceduser.qmd

```diff
@@ -0,0 +1,27 @@
+---
+title: "U.S. Greenhouse Gas Center: Advanced User Notebook"
```

Comment:
I don't think a separate section or page for advanced notebooks is necessary if you see my comment about how to format the data usage notebooks landing page (notebooks for each dataset are introduced in sub-bullets)